homeassistant/helpers/template.py

```diff
@@ -2387,6 +2431,13 @@ def __init__(
         self.globals["max"] = min_max_from_filter(self.filters["max"], "max")
         self.globals["min"] = min_max_from_filter(self.filters["min"], "min")
         self.globals["is_number"] = is_number
+        self.globals["is_list"] = is_list
```

Comment:
What's the reason to have all the `is_*` as globals?

Reply:
it's just a convenience thing so you can do `is_list(x)` or `x | is_list`